databases/pgcli/Portfile

```diff
@@ -35,10 +35,66 @@ if {[variant_isset python34]} {
     python.default_version 27
 }
 
+variant postgresql84 conflicts postgresql90 postgresql91 postgresql92 \
+        postgresql93 postgresql94 postgresql95 postgresql96 \
+        description "Build using postgresql v8.4" {
+            depends_lib-append  port:postgresql84
+}
+
+variant postgresql90 conflicts postgresql84 postgresql91 postgresql92 \
+        postgresql93 postgresql94 postgresql95 postgresql96 \
+        description "Build using postgresql v9.0" {
+            depends_lib-append  port:postgresql90
+}
+
+variant postgresql91 conflicts postgresql84 postgresql90 postgresql92 \
+        postgresql93 postgresql94 postgresql95 postgresql96 \
+        description "Build using postgresql v9.1" {
+            depends_lib-append  port:postgresql91
+}
+
+variant postgresql92 conflicts postgresql84 postgresql90 postgresql91 \
+        postgresql93 postgresql94 postgresql95 postgresql96 \
+        description "Build using postgresql v9.2" {
+            depends_lib-append  port:postgresql92
+}
+
+variant postgresql93 conflicts postgresql84 postgresql90 postgresql91 \
+        postgresql92 postgresql94 postgresql95 postgresql96 \
+        description "Build using postgresql v9.3" {
+            depends_lib-append  port:postgresql93
+}
+
+variant postgresql94 conflicts postgresql84 postgresql90 postgresql91 \
+        postgresql92 postgresql93 postgresql95 postgresql96 \
+        description "Build using postgresql v9.4" {
+            depends_lib-append  port:postgresql94
+}
+
+variant postgresql95 conflicts postgresql84 postgresql90 postgresql91 \
+        postgresql92 postgresql93 postgresql94 postgresql96 \
+        description "Build using postgresql v9.5" {
+            depends_lib-append  port:postgresql95
+}
+
+variant postgresql96 conflicts postgresql84 postgresql90 postgresql91 \
+        postgresql92 postgresql93 postgresql94 postgresql95 \
+        description "Build using postgresql v9.6" {
+            depends_lib-append  port:postgresql96
+}
+
+if {
+    ![variant_isset postgresql84] && ![variant_isset postgresql90] &&
+    ![variant_isset postgresql91] && ![variant_isset postgresql92] &&
+    ![variant_isset postgresql93] && ![variant_isset postgresql94] &&
+    ![variant_isset postgresql95] && ![variant_isset postgresql96]
+} {
+    default_variants    +postgresql96
+}
+
 depends_build       port:py${python.version}-setuptools
 depends_lib-append  port:py${python.version}-click \
                     port:py${python.version}-configobj \
-                    port:py${python.version}-crypto \
```

Comment:
Yeah, I don't see any usage of `crypto` in `pgcli` source code. You probably copied it from `mycli` :)